docs/reference/import.md

```diff
@@ -27,7 +27,7 @@ Unfortunately, copying data is not always feasible for the following reasons:
 3. It requires you to switch to using the lakeFS endpoint in all places at once.
 
 ## Using lakeFS import tool
-To solve this, we offer an import tool which will not copy any data and allow for a more gradual onboarding process.
+To solve this, we offer an import tool that will not copy any data and allow for a more gradual onboarding process.
```

Comment:
Nice.  Mind trying this?
```suggestion
To solve this we offer an import tool that does not copy any data, allowing for a more gradual onboarding process.
```

Reply:
Yeah, I think we can use that.
Thanks for the suggestion!